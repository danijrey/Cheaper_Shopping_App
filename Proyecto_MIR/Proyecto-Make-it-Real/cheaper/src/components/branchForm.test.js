import React from 'react';
import { render, fireEvent, getAllByTestId} from '@testing-library/react'
import BranchForm from './branchForm';



describe ('BranchForm' , ()=> {
    it ('should render correctly', () => {
        const {container} = render (<BranchForm/>);
        expect(container.firstChild).toMatchSnapshot();
    });

    it ('Should render form with data-testid="BranchForm" correctly', ()=>{
        const {getByTestId}=render(<BranchForm/>);
 
        expect(getByTestId("BranchForm")).toBeInTheDocument();
    })
});                             