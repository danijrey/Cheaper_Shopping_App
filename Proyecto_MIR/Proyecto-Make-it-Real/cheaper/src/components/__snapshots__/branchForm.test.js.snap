// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BranchForm should render correctly 1`] = `
<div
  class="App-Branch"
>
  <div
    class="FormContainer"
  >
    <img
      alt="Logo"
      class="NavBar-Logo-Branch"
      src="logo.png"
    />
    <div
      class="PaginaNew-Branch"
    >
      <h1
        class="TitleForm"
      >
        Ingresa los datos de la sucursal
      </h1>
      <div
        class="InputsNew-Branch"
      >
        <form
          class="BranchForm"
          data-testid="BranchForm"
        >
          <input
            class="Branch-Name"
            name="branchName"
            placeholder=" Nombre de sucursal"
            type="text"
            value=""
          />
          <input
            class="Branch-Adress"
            name="branchAdress"
            placeholder=" DirecciÃ³n sucursal"
            type="text"
            value=""
          />
          <button
            class="SubmitBranch"
            type="submit"
          >
            Enviar
          </button>
        </form>
      </div>
    </div>
  </div>
</div>
`;
